proto/zitadel/policy/v2alpha/branding_policy.proto

```diff
@@ -0,0 +1,102 @@
+            example: "\"#CD3D56\"";
+        }
+    ];
+    // hex value for background color
+    string background_color = 4 [
+        (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
+            description: "hex value for background color";
+            example: "\"#FAFAFA\"";
+        }
+    ];
+    // hex value for font color
+    string font_color = 5 [
+        (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
+            description: "hex value for font color";
+            example: "\"#000000\"";
+        }
+    ];
+    // hex value for primary color dark theme
+    string primary_color_dark = 6 [
+        (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
+            description: "hex value for the primary color dark theme";
+            example: "\"#BBBAFA\"";
+        }
+    ];
+    // hex value for background color dark theme
+    string background_color_dark = 7 [
+        (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
+            description: "hex value for background color dark theme";
+            example: "\"#111827\"";
+        }
+    ];
+    // hex value for warning color dark theme
+    string warn_color_dark = 8 [
+        (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
+            description: "hex value for warning color dark theme";
+            example: "\"#FF3B5B\"";
+        }
+    ];
+    // hex value for font color dark theme
+    string font_color_dark = 9 [
+        (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
+            description: "hex value for font color dark theme";
+            example: "\"#FFFFFF\"";
+        }
+    ];
+    bool disable_watermark = 10;
+    string logo_url = 11 [
+        (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
+            description: "hex value for font color dark theme";
+            example: "\"https://acme.com/assets/v1/165617850692654601/policy/label/logo-180950416321494657\"";
+        }
+    ];
+    string icon_url = 12 [
+        (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
+            description: "hex value for font color dark theme";
+            example: "\"https://acme.com/assets/v1/165617850692654601/policy/label/icon-180950498874178817\"";
+        }
+    ];
+    string logo_url_dark = 13 [
+        (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
+            description: "hex value for font color dark theme";
+            example: "\"https://acme.com/assets/v1/165617850692654601/policy/label/logo-dark-180950229376461345\"";
+        }
+    ];
+    string icon_url_dark = 14 [
+        (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
+            description: "hex value for font color dark theme";
+            example: "\"https://acme.com/assets/v1/165617850692654601/policy/label/icon-dark-180950243237405441\"";
+        }
+    ];
+    string font_url = 15;
```

Comment:
```suggestion
    string font_url_light = 15;
```

Reply:
Add attribute `theme: "system" | "light" | "dark"`as we need to distinguish which theme the organization/instance wants to apply.

Reply:
is there a need for this attribute in the current state of the branding? As it is only a Get currently would that not only be necessary in you want to change the branding?

Reply:
It is.. we have some customers who just want to have a certain theme enabled.. This would allow us to implement that.